content/en/docs/Getting started/_index.md

```diff
@@ -9,10 +9,9 @@ description: >
 ## Prerequisites
 
 This package requires;
-- python version 3.8+
-- Anaconda version 4.10+
-
-Are there any system requirements for using this project? What languages are supported (if any)? Do users need to already have any software or tools installed?
+- a linux environment or [docker](https://docs.docker.com/desktop/install/windows-install/) for Windows
+- python version 3.8.10
```

Comment:
Is this a 3.8+ ?
We can't do 3.10 but we can do 3.9

Reply:
ignore

Reply:
lol

Reply:
COAsT really does need 3.8.................. point 10 not just any 3.8